lib/types/plugins.nix

```diff
@@ -89,15 +89,34 @@ with lib; let
     "nvim-dap-ui"
   ];
   # You can either use the name of the plugin or a package.
-  pluginsType = with types;
-    listOf (
-      nullOr (
-        either
-        (enum availablePlugins)
-        package
-      )
+  pluginType = with types;
+    nullOr (
+      either
+      package
+      (enum availablePlugins)
```

Comment:
This has the potential to be a breaking change, did you test it thoroughly?

Reply:
I just extracted `pluginsType` into `pluginsType` + `pluginType`, cuz `extraPlugins.<name>.package` wants a single package instead of a list, I didn't change anything

Reply:
just wanted to make sure it's not referenced in somewhere obscure that we might've missed, but it seems like you've already accounted for that possibility

marking as resolved